packages/contracts/src/interfaces/IStakeManager.sol

```diff
@@ -73,6 +73,30 @@ interface IStakeManager {
         address indexed burnAddress
     );
 
+    /// @notice Emitted when a `devAddress` is changed.
+    event DevAddressSet(
+        address indexed burnAddress
+    );
+
+    /// @notice Emitted if Relay Server is inactive for an `abandonmentDelay` and contract owner initiates its removal.
+    event RelayServerAbandoned(
+        address indexed relayManager,
+        uint256 abandonedTime
+    );
+
+    /// @notice
+    event RelayKeepalive(
```

Comment:
This is also RelayerServer
actually, "Relayer" is shorter (thinking also about all the method below, such as relayAbandoned, etc)

Reply:
I prefer RelayServer. Fixed.